.github/workflows/get-pr-info.yaml

```
@@ -1,14 +1,15 @@
 name: Get info on PR
 on:
-  pull_request_target:
-    branches: [main, ga-staging]
+  pull_request:
```

```suggestion
  pull_request_target:
```
We still need `_target` here, otherwise the workflow will not access to the PSH token secret needed later. 
To limit malicious injections, we're only running this job if PR comes from this repo itself OR the build-fork label has been added so we need to extra cautious when applying that label to PRs from forks.